bin/control

```diff
@@ -377,41 +376,6 @@ function threaddump() {
     fi
 }
 
-function pre_snapshot {
-  start
-  echo "$OPENSHIFT_AEROGEAR_PUSH_TOKEN_KEY" > $OPENSHIFT_DATA_DIR/aerogear_push_token_key
-  cp $OPENSHIFT_AEROGEAR_PUSH_DIR/standalone/configuration/standalone.xml $OPENSHIFT_DATA_DIR/standalone.snapshot.xml
-  stop
-}
-
-function post_snapshot {
-  true
-}
-
-function pre_restore {
-  cleanup_dump
-}
-
-function post_restore {
-  if [ -s $OPENSHIFT_DATA_DIR/aerogear_push_token_key ]; then
-    TOKEN_KEY=$(< $OPENSHIFT_DATA_DIR/aerogear_push_token_key)
-    echo $TOKEN_KEY > $OPENSHIFT_AEROGEAR_PUSH_DIR/env/OPENSHIFT_AEROGEAR_PUSH_TOKEN_KEY
-    sed -i "s/<server socket-binding=\"simplepush\"\(.*\) password=\"[^\"]\+\"/<server socket-binding=\"simplepush\"\1 password=\"$TOKEN_KEY\"/" $WILDFLY_STANDALONE_DIR/configuration/standalone.xml > /dev/null 2>&1
-  fi
-
-  if [ -s $OPENSHIFT_DATA_DIR/standalone.snapshot.xml ]; then
-    cp $OPENSHIFT_DATA_DIR/standalone.snapshot.xml $OPENSHIFT_AEROGEAR_PUSH_DIR/standalone/configuration/standalone_xml_history/snapshot/standalone.snapshot.xml
-  fi
-
-  cleanup_dump
-}
-
-function cleanup_dump {
-  rm -f $OPENSHIFT_DATA_DIR/aerogear_push_token_key
-  rm -f $OPENSHIFT_DATA_DIR/standalone.snapshot.xml
-}
-
-
```

Comment:
Instead of removing all of these hooks, perhaps only the SimplePush parts should be removed? (i.e., lines 382, lines 396-400, and line 410 can be removed) This way, the standalone.xml file will still get backed up when performing snapshot/restore operations so that a user can still refer to it in case he had made any modifications to it.